test/test_manifest.json

```
@@ -9746,5 +9746,13 @@
     "firstPage": 2,
     "lastPage": 2,
     "type": "eq"
+  },
+  {
+    "id": "issue1773°",
```

Please fix the typo:
```suggestion
    "id": "issue17730",
```